server/image_status.go

```diff
@@ -89,6 +89,23 @@ func (s *Server) ImageStatus(ctx context.Context, req *pb.ImageStatusRequest) (r
 	return resp, nil
 }
 
+func (s *Server) createImageInfo(result *pkgstorage.ImageResult) (map[string]string, error) {
```

Comment:
This doesn't need to be a part of `Server` any more as you are no longer calling `isIntermediate` here.